.github/workflows/cli-tests.yaml

```diff
@@ -77,14 +77,19 @@ jobs:
           cache: true
       - name: Build devbox
         run: go install ./cmd/devbox
-      - name: Install additional shells (dash, zsh)
+      - name: Install additional tools and shells (dash, zsh)
         run: |
           if [ "$RUNNER_OS" == "Linux" ]; then
             sudo apt-get update
             sudo apt-get install dash zsh
+            sudo curl -sLo /home/shellspec.tar.gz https://github.com/shellspec/shellspec/releases/download/0.28.1/shellspec-dist.tar.gz
+            tar xvzf /home/shellspec.tar.gz
+            sudo mv shellspec/ /home/
+            sudo ln -s /home/shellspec/shellspec /usr/local/bin/
```

Comment:
This is manual installation for shellspec on ubuntu since it's not in apt-get or snap package managers. I'm open to make this its own step

Reply:
I think putting in own step has a lot of value. It let's us see how long the step takes more and spits up the different parts of the setup.